package.json

```diff
@@ -59,13 +60,16 @@
     "transform": {
       ".(ts|tsx)": "<rootDir>/node_modules/ts-jest/preprocessor.js"
     },
-    "testRegex": "\\.test\\.(ts|tsx)$",
+    "testRegex": "\\.test\\.(ts|tsx|js)$",
```

Comment:
Why do you need to find js files for tests?

Reply:
See comment [here](https://github.com/KnisterPeter/tscodeshift/pull/20#discussion_r142551695)